tests/api/transactions/GET.test.js

```diff
@@ -77,4 +78,14 @@ describe("GET /api/transactions", () =>{
       expect(res.status).toEqual(200);
     });
   });
+  describe("user id does not have a transaction that pertain to it", ()=>{
+    it("should return error object", async() =>{
+      await expect(async() => {
+        await searchQueryTrans();
+      }).rejects.toMatchObject({
+        statusCode: 500,
+        message: "orm tool failed"
+      });
```

Comment:
You should be testing the API endpoint to make sure it returns 500 and this message, not the `searchQueryTrans` function itself.